snuba/consumers/consumer.py

```diff
@@ -601,15 +608,92 @@ class MultistorageKafkaPayload(NamedTuple):
 ]
 
 
+def __message_to_dict(
+    message: Message[KafkaPayload],
+) -> Dict[str, Any]:
+    # TODO: Remove temporary mitigations for DLQ prod test
+    return {
+        "message_payload_value": message.payload.value,
+        "message_offset": message.offset,
+        "message_partition": message.partition.index,
+        "message_topic": message.partition.topic.name,
+    }
+
+
+def skip_kafka_message(message: Message[KafkaPayload]) -> bool:
+    # TODO: Remove temporary mitigations for DLQ prod test
+
+    # expected format is "[topic:partition_index:offset,...]" eg [snuba-metrics:0:1,snuba-metrics:0:3]
+    messages_to_skip = (get_config("kafka_messages_to_skip") or "[]")[1:-1].split(",")
+    return (
+        f"{message.partition.topic.name}:{message.partition.index}:{message.offset}"
+        in messages_to_skip
```

Comment:
I am not sure this will be very useful now that you are catching all exceptions. But if you want to preserve this feature, please identify the message with `partition:offset`, not with the full payload. It will be much easier to manage in terms of config.